doc-content/enterprise-only/openshift/template-deploy-mandatory-proc.adoc

```diff
@@ -0,0 +1,107 @@
+[id='template-deploy-mandatory-{context}-proc']
+= Setting required parameters
+
+// local variables depending on the assembly context
+:params_central!:
+:params_kieserver!:
+:params_kieserver_s2i!:
+:params_kieserver_container_deployment!:
+:params_startup_strategy_first!:
+:params_multideploy!:
+:params_mandatory_maven!:
+:params_serveruser_startupstrategy!:
+
+ifeval::["{context}"=="freeform-monitor"]
+:params_central:
+:params_kieserver:
+:params_mandatory_maven:
+endif::[]
+
+ifeval::["{context}"=="freeform-server-managed"]
+:params_kieserver:
+:params_kieserver_container_deployment:
+:params_startup_strategy_first:
+:params_multideploy:
+:params_mandatory_maven:
+:params_serveruser_startupstrategy:
+endif::[]
+
+ifeval::["{context}"=="freeform-server-immutable"]
+:params_kieserver:
+:params_kieserver_s2i:
+:params_startup_strategy_first:
+:params_multideploy:
+:params_serveruser_startupstrategy:
+endif::[]
+
+
+The following parameters of the template are required in all cases. 
```

Comment:
Need context. Any prereqs?

Reply:
AB4 I think the list of params would be better in a table.

Reply:
Table is not coming in now, because other documents still use a list, we must have some consistency. Marked for discussion for 7.4 restructure.